webapp/plugins/utils.js

```diff
@@ -73,7 +72,10 @@ export const checkMetadata = metadata => {
   // if it doesn't, duplicate from `name`
   if (!displayName) updatedMeta.displayName = name;
   // encode pathName if it exists
-  if (pathName) updatedMeta.pathName = encodeURI(pathName);
+  if (pathName) {
+    assert(pathName !== '/', 'pathName should not be a single forward slash');
```

Comment:
It just crashes the current code execution. This is wrapped in a `try-catch` which is caught below on line 139. This is useful because if a plugin doesn't follow the conventions we need, we basically stop loading it altogether, return an error, and keep going with other plugins.